src/bin/pgcopydb/ld_transform.c

```diff
@@ -2266,18 +2266,31 @@ stream_write_update(FILE *out, LogicalMessageUpdate *update)
 				 * keywords using double quotes, so we should avoid double quoting
 				 * again.
 				 */
-				const char *quoteFormatStr = (old->columns[v][0] == '"') ? "%s%s = $%d" :
-											 "%s\"%s\" = $%d";
-				appendPQExpBuffer(buf, quoteFormatStr,
-								  v > 0 ? " and " : "",
-								  old->columns[v],
-								  ++pos);
-
-				if (!stream_add_value_in_json_array(value, jsArray))
+				if (value->isNull)
```

Comment:
I've 2 minor comments,

1) Could you please add a comment explaining the reason behind this change?
2) I see repetition here in handing the NULL on both insert and update code path, is it possible to move this change into a function and reuse the logic from both places?

Reply:
For the repetition aspect, while I agree it could make sense to refactor that part, I must admit I don't feel like blocking this PR on that: the repetition was there already (just less obvious). I didn't think a comment was needed, seemed obvious to me, but it would be an improvement.

Reply:
I have added a comment.